android/KMAPro/kMAPro/build.gradle

```diff
@@ -12,7 +12,7 @@ apply from: "$rootPath/version.gradle"
 String tier = new File('../../TIER.md').getText('UTF-8').trim();
 
 android {
-    compileSdk 34
+    compileSdk 35
```

Comment:
ack, just realized targetSdkVersion still needs to change to 35. going back to draft